cloud/pkg/cloudstream/config/config.go

```diff
@@ -27,12 +29,41 @@ var once sync.Once
 
 type Configure struct {
 	v1alpha1.CloudStream
+	Ca   []byte
+	Cert []byte
+	Key  []byte
 }
 
 func InitConfigure(stream *v1alpha1.CloudStream) {
 	once.Do(func() {
 		Config = Configure{
 			CloudStream: *stream,
 		}
+
+		ca, err := ioutil.ReadFile(stream.TLSTunnelCAFile)
```

Comment:
What if err !=nil

Reply:
> What if err !=nil

If err!=nil, it means there is no certificate locally, it will reuse cloudhub's certs, This part of code is in tunnelserver.go.